sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -563,7 +535,9 @@ export class MessageSession extends LinkEntity {
       if (receiverError) {
         const sbError = translate(receiverError) as MessagingError;
         if (sbError.code === "SessionLockLostError") {
-          sbError.message = `The session lock has expired on the session with id ${this.sessionId}.`;
+          sbError.message = `The session lock has expired on the session with id ${
+            this.sessionId
+          }.`;
```

Comment:
Nit: Do you have a different formatter or formatter settings than what is in the master branch? This formatting change does not happen on my local set up. Am just concerned the noise the about back & forth will introduce if the different people contributing to the project have different settings..

Reply:
Not on purpose. I'll check - for some reason I can never seem to get the same settings with my prettier plugin.

Reply:
I believe I've got it sorted - I've been opening vscode from the folder (service-bus) and not the root. Prettier is formatting things the same way now through my editor as through 'npm run format'.

Reply:
With that said - _this_ line isn't juggling at all between formats so I think this particular one is "right" if we consider `npm run format` to be the authority here.

Reply:
It appears that this section is formatted "properly" though, at least according to 'npm run format'.